### PR TITLE
Add administrator functionality documentation

### PR DESCRIPTION
**Pull request by PerGon**
Feel free to hold merging as halyard configuration is not done (neither implemented) yet. But provide feedback nonetheless.

I'll be implementing this config in halyard soon and then I'll update the docs accordingly.

### CODE REVIEW

**Comment by ethanfrogers** [comment]
@PerGon I'd think this is okay to merge before the functionality is built into Halyard. At the very least it communicates that the feature is available which is a step above what we had previously.

File: setup/security/admin/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   warning
+   content="This feature gives God Mode like capabilities to the users who are admins. Proceed with caution."
+%}
+
+## Enable and Configure Admin functionality
+
+### Halyard
+TBD
+
+### Manually add configuration in fiat
```
Comment (ethanfrogers) [inline setup/security/admin/index.md]: Small nit, capitalize Fiat.

File: setup/security/admin/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+In the fiat config file, add the following:
```
Comment (ethanfrogers) [inline setup/security/admin/index.md]: Same capitalization fix as above. Also, if we want to merge this before the Halyard work is done you might include a path to the file where this should be added (`~/.hal/$DEPLOYMENT/profiles/fiat-local.yml`).

File: setup/security/admin/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+```yaml
+fiat:
+  admin:
+    roles:
+      - "role of the spinnaker gods"
```
Comment (ethanfrogers) [inline setup/security/admin/index.md]: It might be more useful to have an example of what should actually go here. Something like:
```
- devops-admin
```

**Review by ethanfrogers — APPROVED** [review]
LGTM, just a couple of suggestions!

File: setup/security/admin/index.md (hunk 2 above)
Comment (kuberkaul) [reply setup/security/admin/index.md]: 👍

File: setup/security/admin/index.md (hunk 3 above)
Comment (kuberkaul) [reply setup/security/admin/index.md]: maybe a note on if this using saml service (okta) then these are okta groups and you would need to run `hal deploy apply` after the change.

**Comment by kuberkaul** [comment]
tested the feature and it works pretty well, thanks @PerGon  !

**Comment by PerGon** [comment]
Thanks for the feedback :)
Applied suggestions

**Pull request merged**